sdk/cpp/core/src/common_utilities.hpp

```diff
@@ -0,0 +1,50 @@
+// YANG Development Kit
+// Copyright 2016 Cisco Systems. All rights reserved
+//
+////////////////////////////////////////////////////////////////
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+//////////////////////////////////////////////////////////////////
+
+#ifndef YDK_UTILITIES
+#define YDK_UTILITIES
+
+#include <cstdlib>
+#include <cstring>
+#include <string>
+#include <vector>
+
+#include "logger.hpp"
+#include "service_provider.hpp"
+
+using namespace std;
```

Comment:
Better to avoid having `using namespace` statements in header (`.hpp`) files. It is ok in `.cpp` files. See this link for why: https://stackoverflow.com/a/5849668